Commands/Commands/utils.js

```diff
@@ -79,14 +78,16 @@ commands.stats = {
           inline: true
         }
       )
-      msg.channel.sendMessage('', false, {
-        color: 0x59f442,
-        title: `${msg.author.username} - Statistics`,
-        thumbnail: {
-          url: msg.author.staticAvatarURL
-        },
-        fields: field
-      }).catch(bugsnag.notify)
+      msg.channel.openDM().then((e) => {
+        e.sendMessage('', false, {

```

Comment:
huh, didn't pay attention to what I used for the data, I'd fix it if it actually mattered right now. (rewrite and all)